### PR TITLE
static site bare-bones

### PR DESCRIPTION
**Pull request by slavingia**

### CODE REVIEW

File: build/stylesheets/nvf.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  margin-top: 5px;
+  display: block; }
+  /* line 70, nvf.css.scss */
+  li a {
+    color: #C0392B; }
+
+/* line 75, nvf.css.scss */
+.stately {
+  width: 300px;
+  font-size: 300px;
```
Comment (joshsmith) [inline build/stylesheets/nvf.css]: Can we make slightly larger here, like 500px?

File: build/stylesheets/nvf.css [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+/* line 84, nvf.css.scss */
+.stately .wa,
+.stately .oh,
+.stately .ny {
+  color: #333; }
+
+/* line 90, nvf.css.scss */
+.stately .fl,
+.stately .nc {
+  color: #2980B9; }
```
Comment (joshsmith) [inline build/stylesheets/nvf.css]: Can we use this color for done, `#E0EFF9` for not started, and `#A3D5F7` for in progress?

File: source/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      <li data-state="tx" class="tx">r</li>
+      <li data-state="ut" class="ut">s</li>
+      <li data-state="va" class="va">t</li>
+      <li data-state="vt" class="vt">u</li>
+      <li data-state="wa" class="wa">v</li>
+      <li data-state="wv" class="wv">w</li>
+      <li data-state="wi" class="wi">x</li>
+      <li data-state="wy" class="wy">y</li>
+    </ul>
+    <p>We're <span style="color: #333;">done</span> in three states, and <span style="color: #2980B9;">in progress</span> in two more.
```
Comment (joshsmith) [inline source/index.html.erb]: We're planning on making this a progress bar later, right?

Comment (slavingia) [reply source/index.html.erb]: yes, it looked a little weird with our current progress. once we're farther along, for sure!

(maybe once there's a form and all that jazz in place)

File: build/stylesheets/nvf.css (hunk 1 above)
Comment (slavingia) [reply build/stylesheets/nvf.css]: done

File: build/stylesheets/nvf.css (hunk 2 above)
Comment (slavingia) [reply build/stylesheets/nvf.css]: done

**Review by joshsmith — APPROVED** [review]
👍  LGTM

**Pull request merged**